### PR TITLE
submit

### PR DESCRIPTION
**Pull request by AndrewOluoch**
Created opportunity-divide component in body branch added an image

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -3,10 +3,12 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { OpportunityDivideComponent } from './opportunity-divide/opportunity-divide.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    OpportunityDivideComponent
```
Comment (efoxblack) [inline src/app/app.module.ts:11]: You should have pulled the YearUpSolution component from master so you won't have a conflict in the file here.

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,534 +1 @@
...
-<!-- * * * * * * * * * * and can be replaced. * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * End of Placeholder * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-
-
-
-<router-outlet></router-outlet>
\ No newline at end of file
+<oppertunity-divide></oppertunity-divide>
```
Comment (efoxblack) [inline src/app/app.component.html:1]: This should not be in the app.component.html when merging to master.

File: src/app/opportunity-divide/opportunity-divide.component.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <div class="card-body">
+      
+    <img id="image" src="../../assets/oppertunity-divide.png" alt=""
+    style="width:100%;height:500px;">
+  </div>
+  </div>
+   <!--ACCORDION-->
+
+</body>
+</html>
```
Comment (efoxblack) [inline src/app/opportunity-divide/opportunity-divide.component.html:18]: This closing html tag doesn't need to be here.

**Comment by AndrewOluoch** [comment]
Noted

**Pull request merged**